state.go

```diff
@@ -850,6 +850,16 @@ func (m *Memberlist) aliveNode(a *alive, notify chan struct{}, bootstrap bool) {
 		return
 	}
 
+	if len(a.Vsn) > 3 {
```

Comment:
Is it necessary to check if there are more than 3 items in `a.Vsn`?

```suggestion
	if len(a.Vsn) >= 3 {
```